integration-tests/e2e/main.go

```diff
@@ -45,7 +46,10 @@ func main() {
 
 func createDescFileOfFailedTestcases(failedTestcases []string) string {
 	generatedRunDescPath := filepath.Join(config.TmpDir, "failedTestcasesDescription.txt")
-	file, err := os.OpenFile(generatedRunDescPath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err := os.Remove(kubetest.GeneratedRunDescPath); err != nil {
+		log.Error(err)
```

Comment:
shouldn't we exit if we are unable to remove the file?
not just log the error or is the deletion just a nice to have cleanup?

Reply:
this is only necessary, if e2e.go is executed multiple times, which happens only via local testing.